easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.3-intel-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'PythonBundle'
+
+name = 'netcdf4-python'
+version = '1.5.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://unidata.github.io/netcdf4-python/'
+description = """Python/numpy interface to netCDF."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/Unidata/netcdf4-python/archive/']
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),  # for numpy
+    ('netCDF', '4.7.1'),
+    ('cURL', '7.66.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('cftime', '1.1.1.2', {
+        'checksums': ['35711b5ec3928b9e724817bfa1b7325da205788ee04eae9166cbcd96ea7976a6'],
+    }),
+    (name, version, {
+        'source_urls': ['https://pypi.python.org/packages/source/n/netCDF4'],
+        'source_tmpl': 'netCDF4-%(version)s.tar.gz',
+        'patches': ['netcdf4-python-1.1.8-avoid-diskless-test.patch'],
+        'checksums': [
+            '2a3ca855848f4bbf07fac366da77a681fcead18c0a8813d91d46302f562dc3be',  # netCDF4-1.5.2.tar.gz
```

Comment:
@Darkless012 `netCDF4-1.5.2.tar.gz` should be `netCDF4-1.5.3.tar.gz` in the comment (version fix)